host/discoverd.go

```diff
@@ -105,3 +109,15 @@ func (d *DiscoverdManager) ConnectPeer(ips []string) error {
 	}
 	return err
 }
+
+func (d *DiscoverdManager) UpdateTags(tags map[string]string) error {
+	d.mtx.Lock()
+	defer d.mtx.Unlock()
+	for k, v := range tags {
+		d.inst.Meta[host.TagPrefix+k] = v
```

Comment:
So at a low-level, we could just represent tags that just "exist" with the string `true`, leading to the following:

```
$ flynn-host tags set foo=true # foo gets set to "true"
$ flynn-host tags set foo      # foo gets set to "true"
$ flynn-host tags set foo=     # foo gets deleted
$ flynn-host tags del foo      # foo gets deleted
$ flynn-host tags set foo=bar  # foo gets set to "bar"
```

WDYT?